### PR TITLE
Run two wallet APIs for demo purposes

### LINKED ISSUE #983: Run locally two different wallets for demoing purposes
Make it possible

- [ ] with docker
- [ ] without docker

### PR DESCRIPTION
**Pull request by sveitser**
Close #983

### CODE REVIEW

**Review by philippecamacho — CHANGES_REQUESTED** [review]
Left a couple of comments.
Also should not we update the README.md to explain the tester how to use the two different wallets?

File: bin/wallet-api-bob [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Espresso Systems (espressosys.com)
+# This file is part of the Configurable Asset Privacy for Ethereum (CAPE) library.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+set -eu
+export CAPE_WALLET_PORT=$CAPE_WALLET_PORT_ALICE
```
[the remark this answers is not in the document]
Comment (sveitser) [reply bin/wallet-api-bob:10]: Yes you are right they are inverted. It should still work but I will fix it.

**Comment by sveitser** [comment]
>  Also should not we update the README.md to explain the tester how to use the two different wallets?

Good idea. I added it to the cape-ui README https://github.com/EspressoSystems/cape-ui/pull/252/commits/2fa04fb1b97dbadf6683ca076948b0fd90f896f5

**Comment by philippecamacho** [comment]
It works! I was able to spin up two wallets and send 17 native CAP assets from one to another.

image
image

**Review by philippecamacho — APPROVED** [review]
LGTM

**Comment by smfang** [comment]
@philippecamacho , @sveitser , thanks again for making this feature available!  I ran into  an error with running the `yarn demo`.   (And updated the .env file as  displayed below). I can follow up again next week. 

image

**Pull request merged**